File: pinyougou-parent/pinyougou-user-web/src/main/webapp/js/service/userService.js
app.service('userService',function ($http) {
    this.register=function (code,entity) {
        return $http.post('/user/register.do?code='+code,entity);
    }

    this.createCode=function (phone) {
        return $http.get('/user/createCode.do?phone='+phone);
    }

    this.getInfo=function () {
        return $http.get('/login/getInfo.do');
    }
})

File: pinyougou-parent/pinyougou-user-interface/src/main/java/com/pinyougou/user/service/UserService.java
package com.pinyougou.user.service;

import com.pinyougou.pojo.TbUser;

/**
 * 描述
 *
 * @author 三国的包子
 * @version 1.0
 * @package com.pinyougou.user.service *
 * @since 1.0
 */
public interface UserService {
    void add(TbUser user);


    //生成短信验证码 并发送消息
    void createCode(String phone);


    boolean ischecked(String phone, String code);


}
